README.md

```diff
@@ -9,6 +9,37 @@ A portable library for accurately simulating pokemon battles.
 
 [Documentation](https://pkg.go.dev/github.com/StevensSEC/pokemonbattlelib)
 
+func ExampleBattle() {
```

Comment:
needs to be inside a code block, with the language `go`

See: https://www.markdownguide.org/extended-syntax/#syntax-highlighting